src/client/pages/farm/measures.vue

```diff
@@ -12,25 +14,101 @@
 </template>
 
 <script>
-import { mapState } from 'vuex'
+import { mapGetters, mapState } from 'vuex'
+
+import initMapState from '../../lib/mixins/init-map-state'
+import layerFactory from '../../lib/_mapbox/layer-factory'
+import parcelColors from '../../lib/_mapbox/parcel-colors'
+import unbindFeatureHandlers from '../../lib/mixins/unbind-feature-handlers'
+
 import { MeasuresList } from '../../components'
 import { FooterBar } from '../../components'
 
 export default {
   middleware: [ 'require-parcels' ],
   components: { MeasuresList, FooterBar },
+  mixins: [ initMapState, unbindFeatureHandlers ],
   data() {
     return {
       selectedMeasure: undefined,
     }
   },
   computed: {
+    ...mapState('mapbox/features', ['features']),
     ...mapState('measures', ['measures']),
+    ...mapGetters('measures', ['parcelsPerMeasure'])
   },
   methods: {
-    selectMeasure (measure) {
+    initMapState() {
+      const overlay = layerFactory.parcels()
+      this.$store.dispatch('mapbox/overlays/add', overlay)
+      this.$store.dispatch('mapbox/overlays/setOpacity', { id: overlay.id, opacity: 0.3 })
+
+      this.features.forEach((feature, index) => {
+        this.$store.dispatch('mapbox/features/add', feature)
+        this.$store.dispatch('mapbox/features/addEventHandler', {
+          id: feature.id,
+          event: 'click',
+          handler: (event) => this.selectParcel({ event, index, id: feature.id })
+        })
+        this.$store.dispatch('mapbox/features/setStyle', {
+          id: feature.id,
+          styleOption: 'fill-color',
+          value: parcelColors(feature.properties.gewascategorie)
+        })
+        this.$store.dispatch('mapbox/features/setStyle', {
+          id: feature.id,
+          styleOption: 'fill-opacity',
+          value: 1
+        })
+      })
+    },
+    setFeatureFill({ id, color }) {
```

Comment:
should this method accept a `feature`? makes the places where it's called a lot more readable